webservice/src/services/ApiManager.js

```diff
@@ -144,16 +155,68 @@ export default class ApiManager {
    * @param image
    * @param namespace the namespace where the template should be created
    */
-  createCRDtemplate(
-    course_code,
-    lab_number,
-    description,
-    cpu,
-    memory,
-    image,
-    namespace
-  ) {
-    // TODO: add body
+  createCRDtemplate(namespace, lab_number, description, cpu, memory, image) {
+    return this.apiCRD.createNamespacedCustomObject(
+      this.templateGroup,
+      this.version,
+      namespace,
+      this.templatePlural,
+      {
+        apiVersion: `${this.templateGroup}/${this.version}`,
+        kind: 'LabTemplate',
+        metadata: {
+          name: `${namespace}-lab${lab_number}`,
+          namespace: namespace
+        },
+        spec: {
+          courseName: namespace,
```

Comment:
Same as above about the difference between `namespace` and `courseName`.

Reply:
/saa